Src/ILGPU/Runtime/ExchangeBuffer.cs

```diff
@@ -1,61 +1,49 @@
-﻿// ---------------------------------------------------------------------------------------
-//                                        ILGPU
-//                        Copyright (c) 2016-2020 Marcel Koester
-//                                    www.ilgpu.net
-//
-// File: ExchangeBuffer.cs
-//
-// This file is part of ILGPU and is distributed under the University of Illinois Open
-// Source License. See LICENSE.txt for details
-// ---------------------------------------------------------------------------------------
-
-using ILGPU.Runtime.Cuda;
+﻿using ILGPU.Runtime.Cuda;
 using ILGPU.Runtime.Cuda.API;
 using ILGPU.Util;
 using System;
 using System.Diagnostics;
 using System.Runtime.CompilerServices;
+using System.Diagnostics.CodeAnalysis;
 
 namespace ILGPU.Runtime
 {
     /// <summary>
-    /// A static helper class for the class <see cref="ExchangeBuffer{T}"/>.
+    /// A static helper class for all exchange buffer implementations.
     /// </summary>
     public static class ExchangeBuffer
     {
-        /// <summary>
-        /// Allocates a new exchange buffer that allocates the specified amount of
+                                        /// <summary>
```

Comment:
nit: fix broken summary indentation

Reply:
This was what I was asking about. I messed with the text template, but couldn't get it to work. I was hoping you could point me in the right direction.

Reply:
Ah, nevermind, you told me at the bottom. Thanks!